.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+sudo: false
+node_js:
+#- '2'
+#- '1'
+#- '0.12'
```

Comment:
Yeah I'm going to update these once a version of Fury is released without the legacy stuff.